source/Changelog.txt

```diff
@@ -1,3 +1,6 @@
+20/11/2024 - Dragon Slayer
+	Added SE Npc AI to he following, Bake kitsune, kappa, raiju, yamandon, yomotsu_warrior.
```

Comment:
More NPC AI scripts is always cool! However, these are semi-useless without:
- Script IDs to attach to NPCs - add to jse_fileassociations.scp?
- Actual NPCs to attach them - add DFN entries?